packages/v4/patternfly-docs/content/get-started/design.md

```diff
@@ -5,49 +5,41 @@ section: get-started
 ---
 import './get-started.css';
 
-Whether you are converting from a PatternFly 3 application or starting fresh with the latest and greatest PatternFly design system, this document is intended to help provide designers with information regarding resources and support available.
+Whether you are converting from a preivous PatternFly application or starting fresh with the latest and greatest PatternFly design system, this page is intended to help provide designers with information regarding resources and support available.
 
-## Getting the design kit
-
-The PatternFly design kit was originally developed for [Sketch users](#sketch-users), but is now available for use through [any design tool](#using-other-design-tools).
+## Designing with PatternFly
```

Comment:
I'm like maybe we should introduce the design Kit somewhere instead of going straight to "What is the Design Kit?"

Maybe as one of the "Designing with PatternFly sub sections" and then link to the "What's in the design kit" section from there, and also the "download the design kit" section?

Reply:
I added a little intro of the design kit to this section. I'm not sure if I linked it out correctly

Reply:
This is good, but yes, there is a problem with the link. I think if your remove "/v4/get-started/design" from the target it would work since we are already on that page. Also can't remember if these anchor links are case sensitive. @mmenestr do you know?

Reply:
they are case sensitive! And it seems to be working now :)